src/PageHeader/PageHeader.features.stories.tsx

```diff
@@ -71,14 +71,16 @@ export const WithComponentAsATitle = () => (
     <PageHeader>
       <PageHeader.TitleArea>
         <Breadcrumbs>
-          <Breadcrumbs.Item href="#">...</Breadcrumbs.Item>
-          <Breadcrumbs.Item href="#">primer</Breadcrumbs.Item>
-          <Breadcrumbs.Item href="#">react</Breadcrumbs.Item>
-          <Breadcrumbs.Item href="#">src</Breadcrumbs.Item>
-          <Breadcrumbs.Item href="#">PageHeader</Breadcrumbs.Item>
-          <Breadcrumbs.Item href="#">PageHeader.tsx</Breadcrumbs.Item>
+          <Breadcrumbs.Item href="https://github.com/primer/react/tree/main">react</Breadcrumbs.Item>
+          <Breadcrumbs.Item href="https://github.com/primer/react/tree/main/src">src</Breadcrumbs.Item>
+          <Breadcrumbs.Item href="https://github.com/primer/react/tree/main/src/PageHeader">
+            PageHeader
+          </Breadcrumbs.Item>
+          <Breadcrumbs.Item href="https://github.com/primer/react/blob/main/src/PageHeader/PageHeader.tsx">
+            PageHeader.tsx
+          </Breadcrumbs.Item>
         </Breadcrumbs>
-        <VisuallyHidden as="h2">Visually Hidden Title</VisuallyHidden>
+        <VisuallyHidden as="h2">PageHeader.tsx</VisuallyHidden>
```

Comment:
Should this be `h1` as well?

Reply:
Ah my bad. I just read your replies to @jscholes. I see this is intentional

Reply:
Oh good catch! @pksjce 

Just to clarify, the `h2` intentional ones I mentioned in my response are the ones under the settings page ([reference](https://github.com/github/primer/issues/1115#:~:text=We%20discussed%20the,concerns%20around%20this.))

Looking at the current files page implementation, seems like it has changed since I built this component and stories and it should be h1. However, we will address the cases where breadcrumb components are used as a title in the [following PageHeader PR](https://github.com/primer/react/pull/3816) and the structure will change a bit, so I think we can leave this as is to reduce the number of merge conflicts if that is okay?

Reply:
Yes! That seems fine with me. This one is good to merge 👍